gems/pending/disk/modules/AzureBlobDisk.rb

```diff
@@ -72,7 +72,7 @@ def blob_read(start_byte, length)
     ret = @storage_acct.get_blob_raw(@container, @blob, key, options)
 
     content_md5  = ret.headers[:content_md5].unpack("m0").first.unpack("H*").first
-    returned_md5 = Digest::MD5.hexdigest(ret.body)
+    returned_md5 = Rails.application.config.digest_class.hexdigest(ret.body)
```

Comment:
Interesting.  I thought pending was for things that may go in the next release.  Is there an MD file in the pending directory that I missed?